packages/nodes-base/nodes/Todoist/v2/OperationHandler.ts

```diff
@@ -35,68 +45,174 @@ export interface Command {
 	uuid: string;
 	temp_id?: string;
 	args: {
-		parent_id?: string;
-		id?: number;
-		section_id?: number;
+		parent_id?: number | string;
+		id?: number | string;
+		section_id?: number | string;
 		project_id?: number | string;
 		section?: string;
 		content?: string;
 	};
 }
 
 export const CommandTypes = {
+	// Item/Task commands
 	ITEM_MOVE: 'item_move',
 	ITEM_ADD: 'item_add',
 	ITEM_UPDATE: 'item_update',
 	ITEM_REORDER: 'item_reorder',
```

Comment:
Since this has unused command types, so I have decided to complete it for sake completeness. Or should we clean this up?